V2 design and copy changes

jakep opened the pull request:
**Notable changes**:
- Changed `/mission` to `/about` and updated the design
- Added FAQ to the global header
- Updated design for `/faq`
- Additional $50 donor level
- Miscellaneous copy and design tweaks

kball commented on js/app.js:
```diff
@@ -26,3 +26,10 @@ if(window.mc) {
 }
 
+// Setup FAQ item toggling
+var toggles = document.querySelectorAll('.toggle');
```
FWIW we have jQuery on here, this whole 6 lines of javascript could be just

```
$('.toggle').on('click', function() { $(this).toggleClass('closed'); });
```

This works too though, no big

kball commented:
With the additional elements in the top nav, it no longer fits properly on mobile screens... With the current number we can probably just tweak down the padding and it will still be fine, but we may want to consider moving to e.g. a hamburger icon and off canvas nav

jakep commented:
Included additional changes:
- Mobile bug fixes
- Updated donating power figure
- Tax-deductible copy
- Updated NPO on `/` and `/join`
- Simplify the above line of JS
